pkg/controller/migmigration/backup.go

```diff
@@ -348,3 +455,18 @@ func findPVVerify(pvList migapi.PersistentVolumes, pvName string) bool {
 	}
 	return false
 }
+
+func bytesToSI(bytes int64) string {
```

Comment:
This is cool, does this convert to whatever the appropriate unit is?

The only downside I see to this is that the user may lose the ability to determine if a PVB is still moving if this auto-switches to GB or anything larger.

Reply:
Recommend a comment on this function explaining what it does

Reply:
@djwhatle This comes from the similar logic I used for pvc-migrate. I have not tested this for more than 1 GB. I should. But the formatted string `.2f` should updated the number with a precision of 2 decimal digits : `1.23 GB of 2 GB`. I need to test it though. 

+1 for comment. 

I will add rate calculation in another PR, currently, I am seeing issues with accuracy of the numbers and still need some time to experiment.

Reply:
oh cool. yeah if we are showing 2 decimal places that seems sufficient to me. Thanks!